apps/api/src/domain/games/memory-match/services/memory-match.service.ts

```diff
@@ -124,6 +124,10 @@ export class MemoryMatchService {
         return isInternalError(result) ? this.buildBadUserInputError(result) : result;
     }
 
+    async unpublish(_id: AggregateId): Promise<Error | AggregateId> {
```

Comment:
This should call the corresponding repository method. It should be really similar to the `publish` method on this same class.

Reply:
You've implemented this now.